lib/nodes/Success.ts

```diff
@@ -0,0 +1,15 @@
+import {BlueshellState} from './BlueshellState';
+import {Base} from './Base';
+import {resultCodes as rc} from '../utils/resultCodes';
+
+export class Success<S extends BlueshellState, E> extends Base<S, E> {
+	constructor(
+		name = `Success`
```

Comment:
Why are there backticks here?

Reply:
Copy/pasta from a version that injected a resultcode and actually put it in the name.